lisa/util/process.py

```diff
@@ -90,13 +91,18 @@ def start(
         if shell:
             if not self._is_posix:
                 split_command = ["cmd", "/c", command]
+            elif super_user:
+                split_command = ["sudo", "su", "-c", command]
```

Comment:
What's different with `sudo`?

Reply:
This command need to be run as `root` user. 

Running with `sudo` gives : `-bash: /sys/devices/system/cpu/cpu1/online: Permission denied`.
Running with `sudo su -c` gives : `bash: line 0: echo: write error: Device or resource busy`.

Reply:
did you try with `sudo -sh -c`? the reason may be the command is not support by shell. Some part of command line are not delivered in root shell.

Reply:
This also gives the same result : 
```
lisa@cvm-test:~$ sudo -sh -c echo 0 > /sys/devices/system/cpu/cpu1/online
-bash: /sys/devices/system/cpu/cpu1/online: Permission denied
```

Reply:
Please try in Python code. If test it in the bash, it should be like below. The ssh shell will add quote to make sure each part works. Let me know, if it doesn't work as expected.

`sudo -sh -c "echo 0 > /sys/devices/system/cpu/cpu1/online"`